From 8c279f0bd552dbddcb0efa61334dbc9aadb3a338 Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Tue, 2 Mar 2021 11:06:58 +0700
Subject: [PATCH 113/135] drm: rcar-du: temporary remove set rate for vclk

Currently, vlck and hsclk are hard set. So that temporary remove
set rate for vclk in DSI driver after cpg driver updates set rate
function.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 drivers/gpu/drm/rcar-du/rzg2l_mipi_dsi.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/rcar-du/rzg2l_mipi_dsi.c b/drivers/gpu/drm/rcar-du/rzg2l_mipi_dsi.c
index d22d59a..54ba6bf 100644
--- a/drivers/gpu/drm/rcar-du/rzg2l_mipi_dsi.c
+++ b/drivers/gpu/drm/rcar-du/rzg2l_mipi_dsi.c
@@ -642,8 +642,6 @@ static void rzg2l_mipi_dsi_mode_set(struct drm_bridge *bridge,
 	struct rzg2l_mipi_dsi *mipi_dsi = bridge_to_rzg2l_mipi_dsi(bridge);
 
 	mipi_dsi->display_mode = *adjusted_mode;
-
-	clk_set_rate(mipi_dsi->vclk, adjusted_mode->clock * 1000);
 }
 
 static void rzg2l_mipi_dsi_enable(struct drm_bridge *bridge)
-- 
2.7.4

